pyomo/contrib/doe/examples/updatesuffix_doe_example.py

```diff
@@ -0,0 +1,72 @@
+#  ___________________________________________________________________________
```

Comment:
I fully accept all grumpiness about the two comments I am about to make. In terms of screen reader accessibility, `updatesuffix` isn't the best. There should be an underscore between them: `update_suffix`. I recommend changing the filename to `update_suffix_doe_example.py`